2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,12 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    let fittingArticles  = [];
+    for(let i = 0; i < allArticleTitles.length;  i++){
+        if(allArticleTitles[i].length <= 65){
+            fittingArticles.push(allArticleTitles[i]);
+    }
```

Comment:
This implementation looks good! Just watch out for the indentation to make it more readable.